src/viewer/overlay/index.tsx

```diff
@@ -11,6 +11,9 @@ import {
 } from "../../utils/twitch";
 import Root from "./Root";
 
+ga.create("UA-81509238-9");
```

Comment:
Let's stick with the official template.

```suggestion
ga("create", "UA-81509238-9", "auto");
```